frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/install-lso-sc.tsx

```diff
@@ -108,9 +108,9 @@ export const CreateOCS = withHandlePromise<CreateOCSProps & HandlePromiseProps>(
       <Form className="co-m-pane__body-group">
         <StorageClassSection handleStorageClass={handleStorageClass} filterSC={onlyNoProvSC}>
           <PVsAvailableCapacity
-            replica={getOCSRequestData().spec.storageDeviceSets[0].replica}
+            replica={getOCSRequestData(getName(storageClass)).spec.storageDeviceSets[0].replica}
```

Comment:
Why do you need to pass the storage class name here?

Reply:
This will be solved once `storageClass` param is marked as optional one.

Reply:
I have changed the definition, the device set needs the storage class name to be created.